modules/ai/assets/js/editor/api/index.js

```diff
@@ -45,3 +45,9 @@ export const getImageToImageRemoveText = ( image ) => request( 'ai_get_image_to_
 export const getImagePromptEnhanced = ( prompt ) => request( 'ai_get_image_prompt_enhancer', { prompt } );
 
 export const uploadImage = ( image ) => request( 'ai_upload_image', { ...image } );
+
+export const getHistory = ( type, page, limit ) => request( 'ai_get_history', { type, page, limit } );
```

Comment:
maybe set defaults to `page` and `limit`?

Reply:
Basically, we have defaults in modules/ai/module.php:636, so I thought it might be redundant here. Especially, if one day we'd want to update default values, we'd have to do that in multiple places. Wdyt?